corehq/apps/app_manager/xpath.py

```diff
@@ -177,8 +179,10 @@ class CaseIDXPath(CaseSelectionXPath):
 class CaseTypeXpath(CaseSelectionXPath):
     selector = '@case_type'
 
-    def case(self):
-        return CaseXPath(u"instance('casedb')/casedb/case[%s='%s']" % (self.selector, self))
+    def case(self, instance_name='casedb', case_name='case'):
+        return CaseXPath(u"instance('{inst}')/{inst}/{case}[{sel}='{self}']".format(
+            inst=instance_name, case=case_name, sel=self.selector, self=self
+        ))
```

Comment:
This could just quote the value and call super rather than duplicating the entire function